<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WSAMZN-MVKF1909" failures="3" tests="36" name="com.gapi.testCases.CreateCustomerSupportTicket" time="158.624" errors="1" timestamp="2023-05-19T18:18:41 IST" skipped="4">
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.502"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.676"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.659"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="createCustomerSupportTicketAndVerifyEndToEndFlow" time="6.614">
    <skipped/>
  </testcase> <!-- createCustomerSupportTicketAndVerifyEndToEndFlow -->
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.509"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="retrieveCustomerSupportTicketAttachmentusingRequestId" time="1.006">
    <error message="&#039;org.apache.http.entity.ContentType org.apache.http.entity.ContentType.withParameters(org.apache.http.NameValuePair[])&#039;" type="java.lang.NoSuchMethodError">
      <![CDATA[java.lang.NoSuchMethodError: 'org.apache.http.entity.ContentType org.apache.http.entity.ContentType.withParameters(org.apache.http.NameValuePair[])'
at org.apache.http.entity.mime.MultipartEntityBuilder.buildEntity(MultipartEntityBuilder.java:219)
at org.apache.http.entity.mime.MultipartEntityBuilder.build(MultipartEntityBuilder.java:240)
at org.apache.http.entity.mime.MultipartEntityBuilder$build$5.call(Unknown Source)
at io.restassured.internal.multipart.RestAssuredMultiPartEntity.getEntity(RestAssuredMultiPartEntity.groovy:58)
at io.restassured.internal.multipart.RestAssuredMultiPartEntity.getContentType(RestAssuredMultiPartEntity.groovy:89)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at org.codehaus.groovy.runtime.metaclass.MethodMetaProperty$GetBeanMethodMetaProperty.getProperty(MethodMetaProperty.java:76)
at org.codehaus.groovy.runtime.callsite.GetEffectivePogoPropertySite.getProperty(GetEffectivePogoPropertySite.java:85)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:329)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetPropertySafe(AbstractCallSite.java:448)
at io.restassured.internal.RequestSpecificationImpl$RestAssuredHttpBuilder.shouldApplyContentTypeFromRestAssuredConfigDelegate(RequestSpecificationImpl.groovy:2139)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:194)
at io.restassured.internal.RequestSpecificationImpl$RestAssuredHttpBuilder.doRequest(RequestSpecificationImpl.groovy:2060)
at io.restassured.internal.http.HTTPBuilder.post(HTTPBuilder.java:350)
at io.restassured.internal.http.HTTPBuilder$post$2.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.sendRequest(RequestSpecificationImpl.groovy:1194)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
at groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:166)
at io.restassured.internal.filter.SendRequestFilter.filter(SendRequestFilter.groovy:30)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
at io.restassured.filter.Filter$filter.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.time.TimingFilter.filter(TimingFilter.java:56)
at io.restassured.filter.Filter$filter.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
at io.restassured.filter.Filter$filter$0.call(Unknown Source)
at io.restassured.internal.filter.FilterContextImpl.next(FilterContextImpl.groovy:72)
at io.restassured.filter.FilterContext$next.call(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1686)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
at groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:55)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:203)
at io.restassured.internal.RequestSpecificationImpl.applyPathParamsAndSendRequest(RequestSpecificationImpl.groovy:1692)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:819)
at groovy.lang.GroovyObject.invokeMethod(GroovyObject.java:39)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.call(PogoInterceptableSite.java:45)
at org.codehaus.groovy.runtime.callsite.PogoInterceptableSite.callCurrent(PogoInterceptableSite.java:55)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:203)
at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy:180)
at io.restassured.internal.RequestSpecificationImpl.post(RequestSpecificationImpl.groovy)
at com.gapi.utilities.TestUtilities.postAttachmentRequest(TestUtilities.java:957)
at com.gapi.testCases.CreateCustomerSupportTicket.retrieveCustomerSupportTicketAttachmentusingRequestId(CreateCustomerSupportTicket.java:180)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- retrieveCustomerSupportTicketAttachmentusingRequestId -->
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.647"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="createCustomerSupportTicketAndVerifyEndToEndFlow" time="5.562">
    <failure message="expected [200] but found [500]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at com.gapi.testCases.AddCommentsToTheRemoteHandsTicket.addCommentsToTheExistingTicket(AddCommentsToTheRemoteHandsTicket.java:91)
at com.gapi.utilities.TestUtilities.addCommentToTheRequestAndVerify(TestUtilities.java:473)
at com.gapi.testCases.CreateCustomerSupportTicket.createCustomerSupportTicketAndVerifyEndToEndFlow(CreateCustomerSupportTicket.java:109)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createCustomerSupportTicketAndVerifyEndToEndFlow -->
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.620"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.570"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.752"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.518"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.883"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="0.259"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="createCustomerSupportTicketAndVerifyEndToEndFlow" time="5.492">
    <skipped/>
  </testcase> <!-- createCustomerSupportTicketAndVerifyEndToEndFlow -->
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.833"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="5.068"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="5.572"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.569"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="retrieveCustomerSupportTicketAttachmentusingRequestId" time="1.178">
    <skipped/>
  </testcase> <!-- retrieveCustomerSupportTicketAttachmentusingRequestId -->
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.516"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.623"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.554"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.354"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.703"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.518"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="createCustomerSupportTicketAndVerifyEndToEndFlow" time="5.918">
    <skipped/>
  </testcase> <!-- createCustomerSupportTicketAndVerifyEndToEndFlow -->
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.552"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.810"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.319"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.474"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.804"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCustomerSupportTicket" time="4.791"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="verifyCreateCSTicketForInvalidUser" time="0.292"/>
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="createCustomerSupportTicketAndVerifyEndToEndFlow" time="5.400">
    <failure message="expected [200] but found [500]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at com.gapi.testCases.AddCommentsToTheRemoteHandsTicket.addCommentsToTheExistingTicket(AddCommentsToTheRemoteHandsTicket.java:91)
at com.gapi.utilities.TestUtilities.addCommentToTheRequestAndVerify(TestUtilities.java:473)
at com.gapi.testCases.CreateCustomerSupportTicket.createCustomerSupportTicketAndVerifyEndToEndFlow(CreateCustomerSupportTicket.java:109)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createCustomerSupportTicketAndVerifyEndToEndFlow -->
  <system-out/>
  <testcase classname="com.gapi.testCases.CreateCustomerSupportTicket" name="createCustomerSupportTicketAndVerifyEndToEndFlow" time="5.507">
    <failure message="expected [200] but found [500]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at com.gapi.testCases.AddCommentsToTheRemoteHandsTicket.addCommentsToTheExistingTicket(AddCommentsToTheRemoteHandsTicket.java:91)
at com.gapi.utilities.TestUtilities.addCommentToTheRequestAndVerify(TestUtilities.java:473)
at com.gapi.testCases.CreateCustomerSupportTicket.createCustomerSupportTicketAndVerifyEndToEndFlow(CreateCustomerSupportTicket.java:109)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- createCustomerSupportTicketAndVerifyEndToEndFlow -->
  <system-out/>
</testsuite> <!-- com.gapi.testCases.CreateCustomerSupportTicket -->
